pkg/shell/frames.jsx

```diff
@@ -98,6 +57,28 @@ export const Frames = ({ state, idle_state, hidden }) => {
             return elt;
         }
 
+        function setup_iframe(frame, iframe) {
+            idle_state.setupIdleResetEventListeners(iframe.contentWindow);
+
+            if (iframe.contentDocument && iframe.contentDocument.documentElement) {
+                iframe.contentDocument.documentElement.lang = state.config.language;
+                if (state.config.language_direction)
+                    iframe.contentDocument.documentElement.dir = state.config.language_direction;
+            }
+
+            if (!frame.ready) {
+                frame.ready = true;
+                state.update();
+            }
+        }
+
+        function teardown_iframe(frame, iframe) {
+            if (frame.ready) {
+                frame.ready = false;
+                state.update();
```

Comment:
These 3 added lines are not executed by any test.

Reply:
Hmm.. they should. I'll check.